### ISSUE TITLE
Issue with gradlew on Ubuntu

### ISSUE THREAD

**Issue by Cervator** [MovingBlocks/Terasology#723]
Creating this in response to a report from @glasz on IRC in hoping somebody else can test on various Linux distros / setups to get us more detail. I suspect it may be an issue we have to raise with the Gradle team itself.

Running "./gradlew" with the master branch checked out = OK

Running "./gradlew" with the restructure branch checked out =

~/appz/git/Terasology$ ./gradlew
 ./gradlew: ligne 13: basename : commande introuvable
 ./gradlew: ligne 33: uname : commande introuvable
 ./gradlew: ligne 64: dirname : commande introuvable
 Erreur : impossible de trouver ou charger la classe principale org.gradle.wrapper.GradleWrapperMain

In short the wrapper fails to process the lines in the gradlew file with the ` character wrapping stuff like:

```
APP_BASE_NAME=`basename "$0"`
```

That then makes stuff fail to where Gradle's main class fails to load

@glasz is on Ubuntu 11.04

@skaldarnar can work the same branch just fine - he's on Manjaro (ArchLinux)
